src/doc/src/reference/config.md

```diff
@@ -1098,6 +1098,17 @@ This option is deprecated and unused.
 Specifies the linker which is passed to `rustc` (via [`-C linker`]) when the
 [`<triple>`] is being compiled for. By default, the linker is not overridden.
 
+##### `target.<cfg>.linker`
+This is similar to the [target linker](#targettriplelinker), but using
+a [`cfg()` expression]. 
+
+The config precedence is:
+1. `host.<triple>.linker`, if [host-config](unstable.md#host-config) is enabled.
```

Comment:
I appreciate the effort, though not sure if it is worth a mention, since it may be a bit more confusing that people thought this is a stable feature. OTOH people will be aware of this feature and start testing it.

Therefore, either it is marked as unstable explicitly, or remove.


```suggestion
1. `host.<triple>.linker`, if the unstable feature [host-config](unstable.md#host-config) is enabled.
```

Reply:
I removed it. Thanks!